packages/environment/test/connectOrStartTest.js

```diff
@@ -0,0 +1,33 @@
+const assert = require("chai").assert;
+const Develop = require("../develop");
+
+describe("connectOrStart test network", async function () {
+  const ipcOptions = { network: "test" };
+  const ganacheOptions = {
+    host: "127.0.0.1",
+    network_id: 666,
+    port: 6969
+  };
+
+  it("starts Ganache when no Ganache instance is running", async function () {
+    const connection = await Develop.connectOrStart(ipcOptions, ganacheOptions);
+    assert.isTrue(connection.started);
+    assert.isFunction(connection.disconnect);
+    connection.disconnect();
```

Comment:
You may want to try/finally these, to ensure things shuts down even if the tests fail.

Reply:
Good idea, thanks @davidmurdoch.